<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>webForm</class>
 <widget class="QMainWindow" name="webForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QLineEdit" name="urlWeb">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QWebView" name="webView">
      <property name="url">
       <url>
        <string>about:blank</string>
       </url>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QProgressBar" name="progressBar">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>509</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuWeb">
    <property name="title">
     <string>Web</string>
    </property>
    <addaction name="actionBack"/>
    <addaction name="actionNext"/>
    <addaction name="actionReload"/>
    <addaction name="actionStop"/>
    <addaction name="separator"/>
    <addaction name="actionPrint"/>
    <addaction name="actionPrintPreview"/>
   </widget>
   <addaction name="menuWeb"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Web ToolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionBack"/>
   <addaction name="actionNext"/>
   <addaction name="actionReload"/>
   <addaction name="actionStop"/>
   <addaction name="separator"/>
   <addaction name="actionPrint"/>
   <addaction name="actionPrintPreview"/>
   <addaction name="separator"/>
   <addaction name="actionSpreadLink"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionBack">
   <property name="icon">
    <iconset resource="web.qrc">
     <normaloff>:/web/images/actions/back.png</normaloff>:/web/images/actions/back.png</iconset>
   </property>
   <property name="text">
    <string>back</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset resource="web.qrc">
     <normaloff>:/web/images/actions/forward.png</normaloff>:/web/images/actions/forward.png</iconset>
   </property>
   <property name="text">
    <string>next</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="icon">
    <iconset resource="web.qrc">
     <normaloff>:/web/images/actions/reload.png</normaloff>:/web/images/actions/reload.png</iconset>
   </property>
   <property name="text">
    <string>reload</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="web.qrc">
     <normaloff>:/web/images/actions/stop.png</normaloff>:/web/images/actions/stop.png</iconset>
   </property>
   <property name="text">
    <string>stop</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="web.qrc">
     <normaloff>:/web/images/actions/fileprint.png</normaloff>:/web/images/actions/fileprint.png</iconset>
   </property>
   <property name="text">
    <string>print</string>
   </property>
  </action>
  <action name="actionPrintPreview">
   <property name="icon">
    <iconset resource="web.qrc">
     <normaloff>:/web/images/actions/filefind.png</normaloff>:/web/images/actions/filefind.png</iconset>
   </property>
   <property name="text">
    <string>Print Preview</string>
   </property>
   <property name="toolTip">
    <string>Print Preview</string>
   </property>
  </action>
  <action name="actionSpreadLink">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="web.qrc">
     <normaloff>:/web/images/actions/bookmark_add.png</normaloff>:/web/images/actions/bookmark_add.png</iconset>
   </property>
   <property name="text">
    <string>spreadLink</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="web.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>webView</sender>
   <signal>loadProgress(int)</signal>
   <receiver>progressBar</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>235</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel">
     <x>463</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>webView</sender>
   <signal>titleChanged(QString)</signal>
   <receiver>statusBar</receiver>
   <slot>showMessage(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>378</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
